tests/regression/runner.py

```diff
@@ -28,13 +27,19 @@
 MSG_STATUS_CODE_400 = "{\"message\":\"Invalid request header\"}"
 MSG_STATUS_CODE_404 = "404 Not Found"
 MSG_STATUS_CODE_500 = "500"
+MSG_INVALID_PATH = "{\"message\":\"Invalid path\"}"
+STATUS_CODE_500 = "500"
+STATUS_CODE_405 = "405"
+STATUS_CODE_404 = "404"
+STATUS_CODE_400 = "400"
 EMPTY_REPLY = "000"
 LEN_TAG = 27
 LEN_ADDR = 81
 LEN_MSG_SIGN = 2187
 tryte_alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ9"
 
-if sys.argv[2] == 'Y':
+# the 3rd arg is the option which determine if use the debugging mode of statistical tests
+if sys.argv[3] == 'y' or sys.argv[3] == 'Y':
```

Comment:
Do we really have to take care about both lower and upper case of character `y`?

Reply:
I will remove the upper case one